.github/workflows/kubevious-manifests-ci.yaml

```diff
@@ -48,12 +48,6 @@ jobs:
         with:
           manifests: helm-chart
 
-      - name: Validate istio-manifests
-        id: istio-manifests-validation
-        uses: kubevious/cli@v1.0.61
-        with:
-          manifests: istio-manifests https://github.com/istio/istio/raw/master/manifests/charts/base/crds/crd-all.gen.yaml
-
```

Comment:
This change sounds good to me because we should eventually remove that `istio-manifests` folder.
And users should use [/kustomize/components/service-mesh-istio](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/kustomize/components/service-mesh-istio) instead.